src/styles.css

```diff
@@ -0,0 +1,186 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font: inherit;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+body {
+  width: 417px;
+  height: 718px;
+}
+
+header {
+  width: 288px;
+  padding: 0  20px;
+  display: flex;
+  justify-content: space-between;
+  height: 100px;
+}
+
+header .logo {
+  position: relative;
+  top: 39px;
+  width: 109px;
+  height: 20px;
+  font-family: Poppins, serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 28px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: rgb(96, 112, 255);
+  text-decoration: none;
+}
+
+header .enabled {
+  height: 20px;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  top: 29px;
+  border-radius: 8px;
+}
+
+section {
+  background-image: url("../assets/header-shapes\ mobile.png");
+  background-repeat: not-repeat;
+  width: 100%;
+}
+
+section .container {
+  background-image: url();
+  position: relative;
+  height: 718px;
+  width: 375px;
+  background-repeat: no-repeat;
+  border-bottom-left-radius: 2px;
+  background-size: contain;
+  display: flex;
+  flex-direction: column;
+}
+
+section .container h2 {
+  width: 327px;
+  height: 150px;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  position: relative;
+  margin: 15px;
+  top: 34px;
+}
+
+section .container p:first-of-type {
+  width: 327px;
+  height: 146px;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  position: relative;
+  flex-grow: 0;
+  top: 5px;
+  right: -20px;
+}
+
+section .container p:nth-of-type(2n) {
+  width: 121px;
+  height: 6px;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  flex-grow: 0;
+  position: relative;
+  margin: 15px;
+  top: 5px;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement correct design to your `Heading` and `Paragraph` like a [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=rVw9jLSLBaZEWhxj-0) 🙏🏼 

| Heading Design  | 
|--------------|
```css
font-family: 'Poppins';
font-style: normal;
font-weight: 700;
font-size: 40px;
line-height: 52px;
/* or 130% */

display: flex;
align-items: center;

/* Neutrals/N800 */

color: #172B4D;
 ```  

| Paragraph Design  | 
|--------------|
```css
font-family: 'Poppins';
font-style: normal;
font-weight: 400;
font-size: 16px;
line-height: 24px;
/* or 150% */


/* Neutrals/N600 */

color: #344563;
 ```  
#

![image](https://user-images.githubusercontent.com/63915024/219725861-db686d02-1e15-469b-8920-1f9361daa043.png)

#